app/api/api/views.py

```diff
@@ -113,6 +171,34 @@ def get(self, request, *args, **kwargs):
 
 
 class UserFilterViewSet(GenericAPIView, ListModelMixin):
+    """
+    UserFilterViewSet is a viewset that provides a filtered list of
+    User objects.
+
+    This viewset supports filtering based on the `id` and `is_active`
+    fields of the User model. The `id` field can be filtered using the
+    "in" and "exact" lookup expressions, while the `is_active` field
+    supports the "exact" lookup expression.
+
+    Inherits:
```

Comment:
As before: https://github.com/Health-Informatics-UoN/carrot-mapper/pull/1007#discussion_r2031577237

It would be helpful to not close previous PRs with the comments for features, but just to update them with changes to prevent having to re-review and comment  across them